LoopKitUI/InsulinKit/InsulinDeliveryTableViewController.swift

```diff
@@ -26,6 +26,8 @@ public final class InsulinDeliveryTableViewController: UITableViewController {
 
     @IBOutlet weak var dataSourceSegmentedControl: UISegmentedControl!
 
+    public var disableDeleteAllButton: Bool = false
```

Comment:
[nit] The double negative here is confusing (disable... = false). I suggest `enableDeleteAllButton` rather than `disableDeleteAllButton`.